lib/rucio/web/rest/flaskapi/v1/requests.py

```diff
@@ -71,6 +72,41 @@ def get(self, scope_name, rse):
             return generate_http_error_flask(404, error.__class__.__name__, f'No request found for DID {scope}:{name} at RSE {rse}')
 
 
+class RequestHistoryGet(ErrorHandlingMethodView):
+    """ REST API to get historical requests. """
+
+    @check_accept_header_wrapper_flask(['application/json'])
+    def get(self, scope_name, rse):
+        """
+        List request for given DID to a destination RSE.
+
+        .. :quickref: RequestHistoryGet; list requests
+
+        :param scope_name: data identifier (scope)/(name).
+        :param rse: destination RSE.
+        :reqheader Content-Type: application/json
+        :status 200: Request found.
+        :status 404: Request not found.
+        :status 406: Not Acceptable.
```

Comment:
I don't see the 406 status returned anywhere in the code below

Reply:
Removed.